dotCMS/src/integration-test/java/com/dotcms/enterprise/publishing/remote/bundler/DependencyBundlerTest.java

```diff
@@ -1020,6 +1030,342 @@ public void addLanguageVariableTestCaseInBundle() throws DotSecurityException, D
 
     }
 
+    @DataProvider(format = "%m: %p[0]")
+    public static Object[] configs() throws Exception {
+        return new ModDateTestData[] {
+                new ModDateTestData(false, false, Operation.PUBLISH, true, false),
```

Comment:
Add either the name of each parameter as a comment above so their meaning is clear or use the builder pattern to have named methods instead.

Reply:
a comment would look like :
/ / ModDateTestData(boolean isForcePush , boolean isDownload, false, Operation operation,  boolean shouldExcludeByModDate, boolean addChildDirectly)

Reply:
Let's separate this in various tests to avoid changing the test logic based on the params. It would be removing the last two parameters

Reply:
done https://github.com/dotCMS/core/pull/20293/commits/24854ac1172f4bf79943d66ba236d339d981052a#diff-dcfa32b94c8ab208391686a086d1fa0f4ba566eeff3b1f81c3787e0efa300e8bR1036